eras/conway/impl/cddl-files/conway.cddl

```diff
@@ -34,7 +34,7 @@ header_body =
   , block_body_size  : uint
   , block_body_hash  : $hash32 ; merkle triple root
   , operational_cert
-  , [ protocol_version ]
+  , protocol_version_
```

Comment:
Out of curiosity, is the underscore suffix a standard or commonly used way to name lists in cddl? I haven't encountered it before